### PR TITLE
Add navigation link to QHACK using the QHACK font

### PR DESCRIPTION
**Pull request by josh146**

### CODE REVIEW

File: jinja/navbar.html [hunk 1]
```diff
@@ -2,7 +2,8 @@
     ('/qml', 'qml', 'Quantum machine learning'),
     ('install.html', 'install', 'Install'),
     ('plugins.html', 'plugins', 'Plugins'),
-    ('https://pennylane.readthedocs.io', 'docs', 'Documentation')
+    ('https://pennylane.readthedocs.io', 'docs', 'Documentation'),
```
Comment (co9olguy) [inline jinja/navbar.html:5]: Does not show up for me as a stylized font

Comment (co9olguy) [reply jinja/navbar.html:5]: Can we just put an image here? That way everyone will be able to see easily

**Comment by josh146** [comment]
@co9olguy: @glassnotes very kindly made a black and white version of the QHACK logo for the navbar, let me know what you think!

**Comment by co9olguy** [comment]
LGTM, though I still wonder if we can make a coloured version that stands out better (while not offending @josh146's eyes) :laughing:

**Review by co9olguy — APPROVED** [review]

**Comment by co9olguy** [comment]
@antalszava we will need to update the other pages

**Comment by glassnotes** [comment]
> LGTM, though I still wonder if we can make a coloured version that stands out better (while not offending @josh146's eyes) laughing

We could do a single, flat colour - the issue with the original colour versions was primarily due to the glow effect on the letters. The glow increases the overall size of the image, and when it gets shrunk to only like 20px high, the text itself becomes too small and unclear. Personally though I prefer the black version, the unique font stands out from the other menu items, and in black it doesn't look too out of place.

**Pull request merged**